OPERATING.md

```diff
@@ -0,0 +1,79 @@
+# Operating a Convene Neighborhood
+
+- [Social Responsibilities](#social-responsibilities)
+  - [As a Cooperative](#as-a-cooperative)
+  - [As a Person](#as-a-person)
+  - [As a Legacy Organization](#as-a-legacy-organization)
+- [Economic Model](#economic-model)
+  - [As a Cooperative](#as-a-cooperative-1)
+  - [As a Person](#as-a-person-1)
+  - [As a Classic Corporation](#as-a-classic-corporation)
+- [Technology Details](#technology-details)
+  - [As a Cooperative](#as-a-cooperative-2)
+  - [As a Person](#as-a-person-2)
+  - [As a Classic Non-Profit, Government, or Corporation](#as-a-classic-non-profit-government-or-corporation)
+
+
+Convene is as much a socioeconomic project as it is a technological one. It serves as a platform for designing, building and hosting web software applications that grant super-powers to mutual-aid and mutual benefit solidarity economy organizations.
```

Comment:
`grant super-powers` is too vague.

Reply:
It is purposefully vague; since I can't tell folks what super-powers  their network of client cooperatives need. Again, Convene is an *Operating System*. It's entire value proposition is making it more cost effective for  Producers to provide the appropriate leverage to Consumers.

Reply:
Oooo I like how you said that! ❤️ 

Maybe I can tweak it to this 👇  and update the readme!:

```
Convene is a cost-effective Operating System for building a Community Owned Digital Infrastructure. It provides small businesses/non-profits with the online tools and digital presence needed to run their operations and better serve their customers.

It is a more ethical and affordable alternative to investor-owned online tools like SquareSpace, Etsy, Patreon, Zoom, Yelp, and more.`
```